.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -23,3 +24,37 @@ jobs:
 
           milestone=$(printf "%s\n" "${milestones[@]}" | grep -E "$milestone_selector")
           gh pr edit "$PR_NUMBER" -R "$REPO_NAME" -m "$milestone"
+  set-brave-browser-issues-milestone:
+    runs-on: ubuntu-latest
+    needs: set-pr-milestone
```

Comment:
can this be part of the above job?

Reply:
It can yeah, but I guess it makes more sense as its own separate job. Because it is not a "step" for "setting-pr-milestone". Do you see any benefit of merging them like that ?